cookbook/case_studies/feature_engineering/sqlite_datacleaning/datacleaning_workflow.py

```diff
@@ -0,0 +1,109 @@
+"""
+Data Cleaning Imperative Workflow
+---------------------------------------
+
+This workflow makes use of the feature engineering tasks defined in the other file. 
```

Comment:
We can say that you can simple import the tasks, but we use references because - we are referring to existing code